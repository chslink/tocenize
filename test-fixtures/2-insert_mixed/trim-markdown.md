# aa
# *bb*
# **cc**
# _dd_
# __ee__
# [ff](http://example.org)
# **[gg](http://example.org)**
# hh [![foo](http//example.org/image.png)](http://example.org)