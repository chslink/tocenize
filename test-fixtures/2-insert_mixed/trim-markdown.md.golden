- [aa](#aa)
- [*bb*](#bb)
- [**cc**](#cc)
- [_dd_](#dd)
- [__ee__](#ee)
- [ff](#ff)
- [**gg**](#gg)
- [hh](#hh-)

# aa
# *bb*
# **cc**
# _dd_
# __ee__
# [ff](http://example.org)
# **[gg](http://example.org)**
# hh [![foo](http//example.org/image.png)](http://example.org)